io-engine/src/bin/io-engine-client/v1/pool_cli.rs

```diff
@@ -325,6 +334,92 @@ async fn list(mut ctx: Context, _matches: &ArgMatches) -> crate::Result<()> {
     Ok(())
 }
 
+async fn metrics(mut ctx: Context, _matches: &ArgMatches) -> crate::Result<()> {
+    ctx.v2("Requesting Pool metrics");
+    let response = ctx
+        .v1
+        .stats
+        .get_pool_io_stats(v1rpc::stats::ListStatsOption {
+            name: None,
+        })
+        .await
+        .context(GrpcStatus)?;
+    match ctx.output {
+        OutputFormat::Json => {
+            println!(
+                "{}",
+                serde_json::to_string_pretty(response.get_ref())
+                    .unwrap()
+                    .to_colored_json_auto()
+                    .unwrap()
+            );
+        }
+        OutputFormat::Default => {
+            let stats: &Vec<v1rpc::stats::IoStats> = &response.get_ref().stats;
+            if stats.is_empty() {
+                ctx.v1("No Pool IoStats found");
+                return Ok(());
+            }
+
+            let table = stats
+                .iter()
+                .map(|p| {
+                    let read_latency =
+                        ticks_to_time(p.read_latency_ticks, p.tick_rate);
+                    let write_latency =
+                        ticks_to_time(p.write_latency_ticks, p.tick_rate);
+                    let unmap_latency =
+                        ticks_to_time(p.unmap_latency_ticks, p.tick_rate);
+                    let max_read_latency =
+                        ticks_to_time(p.max_read_latency_ticks, p.tick_rate);
+                    let min_read_latency =
+                        ticks_to_time(p.min_read_latency_ticks, p.tick_rate);
+                    let max_write_latency =
+                        ticks_to_time(p.max_write_latency_ticks, p.tick_rate);
+                    let min_write_latency =
+                        ticks_to_time(p.min_write_latency_ticks, p.tick_rate);
+                    vec![
+                        p.name.clone(),
+                        p.num_read_ops.to_string(),
+                        p.bytes_read.to_string(),
+                        p.num_write_ops.to_string(),
+                        p.bytes_written.to_string(),
+                        p.num_unmap_ops.to_string(),
+                        p.bytes_unmapped.to_string(),
+                        read_latency.to_string(),
+                        write_latency.to_string(),
+                        unmap_latency.to_string(),
+                        max_read_latency.to_string(),
+                        min_read_latency.to_string(),
+                        max_write_latency.to_string(),
+                        min_write_latency.to_string(),
+                    ]
+                })
+                .collect();
+            ctx.print_list(
+                vec![
+                    "NAME",
```

Comment:
nitpick: We can abbreviate `READ` and `WRITE` to `RD` and `WR` which is acceptable, and will make o/p formatting clearer.

Reply:
<img width="1401" alt="image" src="https://github.com/openebs/mayastor/assets/95069770/dfc0eb28-3080-4052-96e8-01215985ee18">

Reply:
I have pasted example Output as screenshot as in PR comment it was not getting properly aligned. Do you want me to use `RD` and `WR`?

Reply:
Suggest you change BYTES_XXX to XXX, and format them using human readable format, ex: 1GiB. Take a look at the test subcommand for wiping on v1 for an example.

Reply:
<img width="1401" alt="image" src="https://github.com/openebs/mayastor/assets/95069770/00781ee5-4232-4678-b0e5-78d86ed6b5eb">
Is this fine?

Reply:
We can abbreviate `READ` as `RD` and `WRITE` as `WR` in all prefixes, which is a fairly acceptable way and o/p will look better formatted.

Reply:
> Nice human units 👍 You mean like `TOTAL_READ` and `TOTAL_WRITE` ? Let others weight in their preferences if they like, I'm ok as is too.

https://github.com/openebs/mayastor/pull/1563#discussion_r1429472685